docs/user/build-customization.rst

```diff
@@ -218,3 +223,36 @@ To setup it, you need to define the version of Node.js to use and install the de
          - npm ci
          # Install any other extra dependencies to build the docs
          - npm install -g jsdoc
+
+
+Override the build process completely
+-------------------------------------
+
+.. warning::
+
+   This feature is in a *beta phase* and could suffer incompatible changes or even removed completely in the near feature.
+   It does not yet support some of the Read the Docs' integrations like the :term:`flyout menu`, search and ads.
+   However, integrating all of them is part of the plan.
+   Use it under your own responsibility.
+
+
+When :ref:`extending the build process <build-customization:extend the build process>` is not enough for a particular use case,
+all the default commands can be overwritten to execute only the commands defined by the project on its config file.
+This can be achieve by using :ref:`config-file/v2:build.commands` key as shown in the following example building a `Pelican <https://blog.getpelican.com/>`_ site:
+
+
+.. code:: yaml
```

Comment:
I feel like we want a minimal example here as well. Perhaps the `echo` example in the PR description, so people understand how it works without knowing pelican.

Reply:
I actually like the Pelican example, but perhaps we could use a more common documentation tool. What I like about this example is it relates the usage to the user -- use `build.commands` if you are using a documentation tool we don't support.

Reply:
Yea, the Pelican example is great. But I think we want something in addition, because otherwise it requires knowing what the pelican command means.

Reply:
I don't have a strong preference for the tooling used here, but I'd like to make it clear in the example _the output folder_. If we hide everything in a `conf.py` file or `pelicanconf.py` with the required settings will be hard to understand what's expected/required from the user.

Reply:
If you have any suggestions that work better than the current one, I'll update the example.